pilot/pkg/serviceregistry/kube/cache.go

```diff
@@ -44,6 +45,9 @@ func newPodCache(ch cacheHandler) *PodCache {
 
 		pod := *obj.(*v1.Pod)
 		ip := pod.Status.PodIP
+
+		log.Printf("Handle pod %s in namespace %s -> %v", pod.Name, pod.Namespace, pod.Status.PodIP)
```

Comment:
This is too verbose.

Reply:
It's still one line - but it's very useful to know what is happening ( I spent quite a bit of time figuring out which events have a real pod and which not ).  It's not a very frequent event.